src/components/navbar/Navbar.js

```diff
@@ -44,6 +66,14 @@ const Navbar = ({ clickedLogin, clickedSignup, logOut }) => (
           </Menu>
         ) : (
           <Menu mode="horizontal" style={{ float: 'right' }} data-test="unauthenticated-menu">
```

Comment:
Remove inline style

Reply:
For the common styles across several files, you can create a shared folder and a scss file inside it. Then extract those styles into that file.

Reply:
Done